src/main/java/gregtech/common/metatileentities/multi/electric/generator/LargeTurbineWorkableHandler.java

```diff
@@ -8,12 +8,15 @@
 import gregtech.api.metatileentity.multiblock.RecipeMapMultiblockController;
 import gregtech.api.recipes.Recipe;
 import gregtech.api.recipes.RecipeBuilder;
+import gregtech.api.util.TextFormattingUtil;
```

Comment:
The additional methods to TextFormattingUtil are needed for `"%,d"` formatting to add commas for numbers, otherwise the lang fails to apply. It's possible there's some method of TextComponantTranslation or something that could be used to resolve this issue, removing the need for the additional methods in TextFormattingUtil.

Reply:
I found another way to do it, resolved with 69a0dae